app/views/audits/index.html.erb

```diff
@@ -1,3 +1,13 @@
+ <%= if @host
+   breadcrumbs do
+     [
+       {caption: _("Hosts"), url: (url_for(hosts_path) if authorized_for(hash_for_hosts_path))},
+       {caption: @host.name, url: (host_path(@host) if authorized_for(hash_for_host_path(@host)))},
+       {caption: _('Audits'), url: url_for(audits_path)},
```

Comment:
It should imho be just `Hosts > %{host_name} > Audits`:
```ruby
[
  {caption: _("Hosts"), url: (url_for(hosts_path) if authorized_for(hash_for_hosts_path))},
  {caption: @host.name, url: (host_path(@host) if authorized_for(hash_for_host_path(@host)))},
  {caption: _('Audits')},
]
```

Having a link to listing all audits nested under hosts doesn't make much sense to me.